tsdb/engine.go

```diff
@@ -60,6 +74,21 @@ func NewEngine(path string, walPath string, options EngineOptions) (Engine, erro
 	// Only bolt-based backends are currently supported so open it and check the format.
 	var format string
 	if err := func() error {
+		// if it's a dir then it's a tsm1 engine
```

Comment:
Is there any other way to check that it's a tsm1 engine? What happens if we ever add another engine (`tsm2`?) and it's also creates a directory at this point? How will we distinguish in the future?